### PR TITLE
Debugging and coding style

### PR DESCRIPTION
**Pull request by sashko**

### CODE REVIEW

File: mediaplayerbackend.cpp [hunk 1]
```diff
@@ -340,12 +338,12 @@ void MediaPlayerBackend::onTracklistChanged()
 
 void MediaPlayerBackend::onTimePositionReceived(int timePosition)
 {
-	qDebug() << "onTimePositionReceived";
+    qDebug() << "onTimePositionReceived";
```
Comment (JEderonn) [inline mediaplayerbackend.cpp]: add print of timePosition to qDebug

**Review by JEderonn — CHANGES_REQUESTED** [review]
Why not improve debugging for:

- fetchData
- onStateReceived
- onPlaybackStateChanged
- onTracklistChanged
- onTracksReceived

The events related to tracklist and state changes are needed.

In general, add the variable in all calls to qDebug(). There is no need to avoid having them, only if there is a printout of a huge amount of data.

File: mediaplayerbackend.cpp [hunk 2]
```diff
@@ -208,31 +210,36 @@ void MediaPlayerBackend::setCurrentIndex(int index)
...
 
     case QIviMediaPlayer::RepeatAll:
+        qDebug() << "setPlayMode: RepeatAll";
         m_tracklistController.setSingle(false);
         m_tracklistController.setRepeat(true);
         m_tracklistController.setRandom(false);
         break;
 
     case QIviMediaPlayer::Shuffle:
+        qDebug() << "setPlayMode: Shuffle";
```
Comment (JEderonn) [inline mediaplayerbackend.cpp:256]: pl add default case

File: mediaplayerbackend.cpp [hunk 3]
```diff
@@ -270,21 +277,23 @@ void MediaPlayerBackend::onCurrentTlTrackReceived(const mopidy::TlTrack &tlTrack
...
 
     case mopidy::PlaybackState::Playing:
+        qDebug() << "onStateReceived: Playing";
         m_playbackState = QIviMediaPlayer::Playing;
         m_playbackController.getCurrentTlTrack();
         m_playbackController.getTimePosition();
         break;
 
     case mopidy::PlaybackState::Stopped:
+        qDebug() << "onStateReceived: Stopped";
```
Comment (JEderonn) [inline mediaplayerbackend.cpp:314]: pl add default case, and print qDebug() + the value

File: mediaplayerbackend.cpp [hunk 4]
```diff
@@ -297,20 +306,22 @@ void MediaPlayerBackend::onStateReceived(mopidy::PlaybackState state)
...
 
     case mopidy::PlaybackState::Playing:
+        qDebug() << "onPlaybackStateChanged: Playing";
         m_playbackState = QIviMediaPlayer::Playing;
         m_playbackController.getCurrentTlTrack();
         m_playbackController.getTimePosition();
         break;
 
     case mopidy::PlaybackState::Stopped:
+        qDebug() << "onPlaybackStateChanged: Stopped";
```
Comment (JEderonn) [inline mediaplayerbackend.cpp:346]: pl add default value

**Review by JEderonn — APPROVED** [review]
approved after fixing the minor comment regarding missing default case

File: mediaplayerbackend.cpp (hunk 2 above)
Comment (sashko) [reply mediaplayerbackend.cpp:256]: what would default case do?

File: mediaplayerbackend.cpp (hunk 3 above)
Comment (sashko) [reply mediaplayerbackend.cpp:314]: what would default case do? value of what?

File: mediaplayerbackend.cpp (hunk 4 above)
Comment (sashko) [reply mediaplayerbackend.cpp:346]: what would default case do?

File: mediaplayerbackend.cpp (hunk 2 above)
Comment (JEderonn) [reply mediaplayerbackend.cpp:256]: For example:
default:
  print qWarning() << "setPlayMode: error, playMode not supported: " << playMode;
  break;

File: mediaplayerbackend.cpp (hunk 3 above)
Comment (JEderonn) [reply mediaplayerbackend.cpp:314]: For example:
default:
  print qWarning() << "onStateReceived: error, state not supported: " << state;
  break;

File: mediaplayerbackend.cpp (hunk 4 above)
Comment (JEderonn) [reply mediaplayerbackend.cpp:346]: For example:
default:
  print qWarning() << "onPlaybackStateChanged: error, state not supported: " << state;
  break;

**Pull request merged**